dronegen/common.go

```diff
@@ -15,3 +22,36 @@ var (
 		Path: "/tmpfs",
 	}
 )
+
+// getMakefileTarget gets the correct Makefile target for a given arch/fips combo
+func getMakefileTarget(params buildType) string {
+	makefileTarget := fmt.Sprintf("release-%s", params.arch)
+	if params.fips {
+		makefileTarget += "-fips"
+	}
+	return makefileTarget
+}
```

Comment:
this probably belongs in `push.go`

I intended this file to contain vars/functions re-used by multiple files (e.g. steps to checkout code, or common volumes like tmpfs and docker).
If a function is specific to a single set of pipelines (like `push-` ones), put it as close as possible to the caller

Reply:
My intention was to use this function in both `push.go` and `build.go` when I start migrating the next set of pipelines; I might be moving it a bit early.